# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libxfont/rpath_deps.patch.cross
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./xfont.pc.in.orig	2010-05-03 18:36:28.000000000 -0400
+++ ./xfont.pc.in	2010-05-03 18:37:57.000000000 -0400
@@ -6,8 +6,8 @@
 Name: Xfont
 Description: X font Library
 Version: @VERSION@
-Requires: xproto fontsproto
-Requires.private: fontenc @FREETYPE_REQUIRES@
+Requires: xproto fontsproto fontenc
+Requires.private: @FREETYPE_REQUIRES@
 Cflags: -I${includedir}
 Libs: -L${libdir} -lXfont
 Libs.private: -lz -lm
